.github/workflows/release.yml

```diff
@@ -63,7 +63,7 @@ jobs:
         git config --global user.name "User"
         ./y.sh prepare
         # FIXME(antoyo): we cannot enable LTO for stdarch tests currently because of some failing LTO tests using proc-macros.
-        echo -n 'lto = "fat"' >> build_sysroot/Cargo.toml
+        echo -n 'lto = "fat"' >> build/build_sysroot/Cargo.toml
```

Comment:
Could the CI failure due to the fact that this change is overwritten by the fact that you now copy this file?

Reply:
Hum indeed.